README.md

```diff
@@ -149,6 +149,30 @@ You can update multiple docs versions at the same time because each directory in
 
 Example: When you change any file in `versioned_docs/version-0.7.0/`, it will only affect the docs for version `0.7.0`.
 
+## Configs
+Configs can be automatically updated by following these steps.
+
+Execute these from hudi-utils dir <br/>
+Ensure you have hudi artifacts from latest master has been installed in your local maven repo before proceeding. <br/>
```

Comment:
`Ensure you have hudi artifacts from latest master has been installed`
-> `Ensure you have hudi artifacts from latest master installed`